package.json

```diff
@@ -47,7 +47,7 @@
     "coveralls": "~2.11.8",
     "istanbul": "~0.4.2",
     "level-test": "~1.7.0",
-    "leveldown": "^1.4.4",
+    "leveldown": "~1.4.4",
```

Comment:
^ resolves to 1.5.3 which errors as mentioned [here](https://github.com/mcollina/levelgraph-jsonld/pull/37#issuecomment-271086719)